pkg/operators/ebpf/converters.go

```diff
@@ -106,10 +112,63 @@ func (i *ebpfInstance) initEnumConverter(gadgetCtx operators.GadgetContext) erro
 	return nil
 }
 
+func (i *ebpfInstance) initStackConverter(gadgetCtx operators.GadgetContext) error {
+	kernelSymbolResolver, err := kallsyms.NewKAllSyms()
```

Comment:
For performance, could we skip the call to `kallsyms.NewKAllSyms()` if there are no fields with `StackIdKernelType`?